data/test_nlu_data/nlu.md

```diff
@@ -0,0 +1,50 @@
+## intent:greet
```

Comment:
do we need to duplicate this, seems to be an exact copy of the moodbot data

Reply:
you're right, we can just use that file